packages/discussions/src/types.ts

```diff
@@ -705,6 +705,55 @@ export interface ISearchChannelsOptions extends IHubRequestOptions {
   params?: ISearchChannels;
 }
 
+/**
+ * Role types
+ *
+ * @export
+ * @enum {string}
+ */
+export enum Role {
+  READ = "read",
+  WRITE = "write",
+  READWRITE = "readWrite",
+  MODERATE = "moderate",
+  MANAGE = "manage",
+  OWNER = "owner",
+}
+
+/**
+ * permission object that will populate ACL interface
+ *
+ * @export
+ * @interface IPermission
+ */
+export interface IPermission {
+  role: Role;
+  createdAt: string;
+  updatedAt: string;
+  accessibleAfter: string;
+}
+
+/**
+ * request options for creating channel with ACL
+ *
+ * @export
+ * @interface IWithACL
+ * @extends {IWithSettings}
+ */
+export interface IWithACL extends IWithSettings {
```

Comment:
Would the name `ICreateChannelWithACL` be a more accurate? i.e., it extends IWithSettings, which looks specific to Channels, so if this interface is only used for channels, let's include that in the name.

Reply:
Yea, I agree